Source/WTF/wtf/CallbackAggregator.h

```diff
@@ -32,19 +32,27 @@
 
 namespace WTF {
 
-class CallbackAggregator : public ThreadSafeRefCounted<CallbackAggregator> {
+template <DestructionThread destructionThread>
+class CallbackAggregatorOnThread : public ThreadSafeRefCounted<CallbackAggregatorOnThread<destructionThread>> {
```

Comment:
The idea was to pass destructionThread as template parameter to ThreadSafeRefCounted.